src/qt/askpassphrasedialog.cpp

```diff
@@ -90,7 +91,7 @@ void AskPassphraseDialog::setModel(WalletModel *_model)
 void AskPassphraseDialog::accept()
 {
     SecureString oldpass, newpass1, newpass2;
-    if(!model)
+    if(!model && mode != Encrypt)
```

Comment:
```suggestion
    if (!model && mode != Encrypt)
```
also, move the body (`return;`) onto the same line or enclose in brackets

Reply:
Changed the spacing, leaving `return` as is because it was not touched.